share/completions/adb.fish

```diff
@@ -182,8 +174,8 @@ complete -n '__fish_seen_subcommand_from reconnect' -c adb -x -a device -d 'Kick
 
 # commands that accept listing device files
 complete -n '__fish_seen_subcommand_from shell' -c adb -f -a "(__fish_adb_list_files)" -d 'File on device'
-complete -n '__fish_seen_subcommand_from pull' -c adb -f -a "(__fish_adb_completion_for_pull)" -d 'File on device'
-complete -n '__fish_seen_subcommand_from push' -c adb -f -a "(__fish_adb_completion_for_push)" -d 'File on device'
+complete -n '__fish_seen_subcommand_from pull' -c adb -F -a "(__fish_adb_list_files)" -d 'File on device'
+complete -n '__fish_seen_subcommand_from push' -c adb -F -a "(__fish_adb_list_files)" -d 'File on device'
 
 # logcat
 complete -n '__fish_seen_subcommand_from logcat' -c adb -f
```

Comment:
Do you know what this line is completing?

Reply:
Yes, completes host files and device files at the same time.  Found it hard to detect when to complete second parameter because first allows multi.

`adb push [local]... [remote]`
`adb pull [remote]... [local]`

Reply:
Sorry, GitHub highlighted all 5 lines but I only meant the last one (after `# logcat`).

Reply:
It means not to complete file for `adb logcat`.

Reply:
Ok, thanks. This all looks good, thanks for explaining the adb logic to me and for bearing through the requested changes and refactors. If you’re happy with the completions for now, I can squash and merge.